docs/interacting-with-openshift.md

```diff
@@ -99,48 +99,58 @@ $ minishift logs
 <a name="updating-openshift-configuration"></a>
 ## Updating OpenShift configuration
 
-Once OpenShift is running, you can view and change the master and
-node configuration of your OpenShift cluster.
+While OpenShift is running, you can view and change the master or the node configuration of your OpenShift cluster.
```

Comment:
Since, we mentioned `OpenShift` at the beginning of sentence, I think we can omit during the sentence like here `configuration of your OpenShift cluster`.
It can be `configuration of your cluster`.